src/pages/Home/Home.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from "react-router-dom";
+import DefaultWithoutSearch from "../../layouts/DefaultWithoutSearch";
+import styles from "./Home.module.css";
+import banner1 from "./Banner1.jpg";
+import banner2 from "./Banner2.jpg";
+import signinBanner from "./SigninBanner.jpg";
+import allCategoryBanner from "./AllCategoryBanner.jpg";
+import { useProducts } from "../../contexts/ProductsProvider";
+import { useState } from "react";
+
+const Home = () => {
+  const { isProductsLoading, categories } = useProducts();
+  const styleObj = [
+    { position: "topRight", image: banner1 },
+    { position: "bottomLeft", image: banner2 },
+    { position: "bottomRight", image: signinBanner },
+  ];
+
+  if (isProductsLoading) {
+    return (
+      <DefaultWithoutSearch>
```

Comment:
`<DefaultWithoutSearch>` this is repeating. Can we place it in the parent itself?